### ISSUE TITLE
A11Y Lab - Sensory Characteristics

### ISSUE THREAD

**Issue by glenn-sorrentino** [scidsg/hushline#158]
https://github.com/scidsg/project-info/blob/main/hush-line/1.%20Accessibility/WCAG%202.1%20AAA_HushLine%20website.pdf

1.3.3.Sensory Characteristics: Instructions provided for understanding and
operating content do not rely solely on sensory characteristics of
components such as shape, size, visual location, orientation, or sound.
The intent of this Success Criterion is to ensure that all users can access instructions for
using the content, even when they cannot perceive shape or size or use information
about spatial location or orientation.
While using shape, provide visible label/name to the control

![Screenshot 2023-10-19 at 12 00 16 PM](https://github.com/scidsg/hushline/assets/28545431/602f573a-77eb-4711-b254-072870167b48)
